### ISSUE TITLE
Stop sending spam and adding me to mailing lists without my consent

### ISSUE THREAD

**Issue by emrox** [fyne-io/fyne#3621]
### Checklist

- [X] I have searched the [issue tracker](https://github.com/fyne-io/fyne/issues) for open issues that relate to the same feature, before opening a new one.
- [X] This issue only relates to a single feature. I will open new issues for any other features.

### Is your feature request related to a problem?

I just received an email to my GitHub assigned address I never use for any subscriptions promoting adding a badge to my profile. It also states I can unsubscribe from a newsletter I never subscribed to. Please stop that. Just because I star your project it doesn't mean I consent to any communication / newsletter.

### Describe the solution you'd like to see.

Stop sending unrequested newsletters and adding people to newsletters without their consent

**Comment by andydotxyz** [comment]
I'm really sorry you were not happy to get the email. I was looking to get more feedback on project direction.
Clearly this was not the best way to go about it...

**Comment by andydotxyz** [comment]
It seems that some folk might have been emailed accidentally - conversation on #3622.
My apologies again for the trouble, closing this issue as duplicate of the other.

**Comment by Ichicoro** [comment]
Yeah, I'd like to not get emailed either. It's not the first time a developer uses their starred list to send mass emails.
Avoid doing this stuff, please :)

**Comment by andydotxyz** [comment]
I appreciate the feedback and apologise for the hassle of an unwanted email.

Out of interest is there a way that you have seen developers engage their community in such a discussion that you would recommend?

**Comment by alx3dev** [comment]
> I appreciate the feedback and apologise for the hassle of an unwanted email.
> 
> Out of interest is there a way that you have seen developers engage their community in such a discussion that you would recommend?

Making a simple chat page on your web-site would help a lot. I found a chat.fyne.io, but there's no https and also require microphone and camera (which is a big no-no for many people). I think Fyne require a lot more documentation, and I would like to help with that, but it also require me to have an easy way to contact you (instead of opening new discussion for simple question).

**Comment by andydotxyz** [comment]
Thanks so much for the thoughts. The chat page you found was a basic front-end to "Jitsi" that we use for our monthly contributor calls.
We are also on Slack and Discord if that would be suitable? Otherwise I could investigate a web based chat, but it will be a little tricky as it would lack the community contributions that our social channels have.
